### DISCUSSION TITLE
Double tray icons in TaskBar ( KDE PLASMA ) , how to combine them ?

### DISCUSSION THREAD

**Discussion by KAGEYAM4** [Martchus/syncthingtray#188] [Q&A] [answered]
Other than going into `Configure system tray -> Entries -> Syncthing Tray -> Alway Hidden`, is there alternative. Because is `Syncthing-Plasmoid` there is no `quit` option whereas in `Syncthing-tray` there is one.

![Screenshot_20230612_112524](https://github.com/Martchus/syncthingtray/assets/75798544/558b38dd-5a9e-4e41-8dcf-c7a698244a0e)

**Comment by Martchus** [comment, marked as answer]
There is no way to combine them. You need to decide which one you want and which one to get rid of.

To get rid of "Syncthing Tray", go into its settings and disable the auto start. Then simply close it via its context menu.

To get rid of "Syncthing Plasmoid" shown as part of the system tray plasmoid, right-click on the arrow icon of the system tray plasmoid and open its settings. Under entries, deactivate Syncthing.

To get rid of "Syncthing Plasmoid" shown as stand-alone plasmoid, enter Plasma's edit mode and simply delete the plasmoid.

I suppose I should add this information to the README.

**Reply by KAGEYAM4** [reply]
Thank You.
